README.md

```diff
@@ -3,23 +3,20 @@ Common Travis CI configuration for MoveIt! project
 
 Authors: Dave Coleman, Isaac I. Y. Saito, Robert Haschke
 
-- Uses Docker for all Distros
-  - Travis does not currently support Ubuntu 16.04
-  - Based on OSRF's pre-build ROS Docker container to save setup time
-  - Uses MoveIt's pre-build Docker container to additionally save setup time
-- Simple - only contains features needed for MoveIt!
-- Clean Travis log files - looks similiar to a regular .travis.yml file
-- Runs tests for the current repo, e.g. if testing moveit\_core only runs tests for moveit\_core
+- Uses [pre-build Docker containers](https://hub.docker.com/r/moveit/moveit) for all ROS distros to save setup time
+- Simple Travis configuration
```

Comment:
The above .travis.yml file is very confusing to me, yet I've spent many hours developing moveit_ci in the past. please keep in mind other, far less familiar developers trying to use and maintain these scripts in the future and don't make things too complicated!

Reply:
What is confusing with the example [`.travis.yml`](https://github.com/ros-planning/moveit_ci/blob/88447c8c27b459540ad8275065ec169a6668df60/README.md)? It's an extended and simplified version of current MoveIt's [`.travis.yml`](https://github.com/ros-planning/moveit/blob/405557df02a06e19dd4dcfa0ee4a0a8472fd34ef/.travis.yml).
Simplified in the sense that common variables are now defined in the `global` `env` section.
Extended, because we need to document the new testing capabilities: clang-format, clang-tidy

I agree, though that the [`.travis.yml` of `moveit_ci`](https://github.com/ros-planning/moveit_ci/blob/88447c8c27b459540ad8275065ec169a6668df60/.travis.yml)`  was overwhelming. I already restructured the unit tests to simplify them. However, before we didn't had any unit tests.

> ... don't make things too complicated!

The code in the travis.sh script has not changed very much and if so, only simplified by introducing helper functions with speaking names.

> I've spent many hours developing moveit_ci in the past.

I appreciate your original work on this and I was involved at that time as well.
Conversely, please also appreciate my effort in further improving those scripts. I also spent many hours over my weekend.